tests/mocks/resmoke.py

```diff
@@ -0,0 +1,59 @@
+"""Mock of resmoke.py for testing task generation."""
```

Comment:
This file is a mocked return value for resmoke.py? Like in our other project, something like `mock_git.return_value=abc`?

Reply:
This file is just a fake version of resmoke.py. You can call it with various options and it will return fake data. Since the tool gathers various information from resmoke, we need some way to return fake data for the integration tests. This file lets us do that. We point the tool at this file instead of a real instance of resmoke to gather the data it needs.